test/unit/Cover/burnStake.js

```diff
@@ -26,13 +26,13 @@ describe.skip('burnStake', function () {
 
     const {
       internalContracts: [internal1],
-      emergencyAdmin,
+      // emergencyAdmin,
     } = this.accounts;
 
     const { productId, coverAsset, period, amount, targetPriceRatio } = coverBuyFixture;
 
-    await cover.connect(emergencyAdmin).enableActiveCoverAmountTracking([], []);
-    await cover.connect(emergencyAdmin).commitActiveCoverAmounts();
+    // await cover.connect(emergencyAdmin).enableActiveCoverAmountTracking([], []);
```

Comment:
is this left in because wwe're not sure of what to do here yet?

For buckets vs expiration

Reply:
removed as they were not needed